mmv1/third_party/terraform/website/docs/d/compute_snapshot.html.markdown

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "Compute Engine"
+page_title: "Google: google_compute_snapshot"
+description: |-
+  Get information about a Google Compute Snapshot.
+---
+
+# google\_compute\_snapshot
+
+Get information about a Google Compute Persistent disks.
```

Comment:
This seems like a copy-paste error, should it refer to snapshots?

Reply:
Fixed.